README.md

```diff
@@ -140,11 +139,23 @@ sudo apt-get install -y \
         file \
         zlib1g-dev
 ```
+
+On Ubuntu 20.04, install gcc-10 which has C++20 support:
+```bash
+sudo apt-get install -y gcc-10 g++-10 cpp-10
```

Comment:
`g++-10` is all you need, since it depends on `gcc-10`, which also then depends on `cpp-10`